src/content/docs/reference/Uniforms/Matrices/gbufferModelView.mdx

```diff
@@ -4,9 +4,11 @@ description: gbufferModelView
 sidebar:
   label: gbufferModelView
   order: 1
-  badge:
-    text: Unfinished
-    variant: caution
 ---
 
-### `uniform ;`
\ No newline at end of file
+### `uniform mat4 gbufferModelView;`
+
+Transforms coordinates from player space to view/eye space.
+```glsl
+vec3 view_pos = mat3(gbufferModelView) * eye_player_pos;
```

Comment:
This is a bit picky, but feet player space is used as much if not more than eye player space, and the matrix includes the offset for that conversion so it makes sense to put both.

```suggestion
vec3 view_pos = gbufferModelView * vec4(feet_player_pos, 1.0);
vec3 view_pos = mat3(gbufferModelView) * eye_player_pos;
```